gen/internal/tests/thrift/enums.thrift

```diff
@@ -52,5 +52,13 @@ enum lowerCaseEnum {
     containing, lower_case, items
 }
 
+// EnumWithLabel use label name in serialization/deserialization
+enum EnumWithLabel {
+    username (go.label = "surname"),
+    password (go.label = "hashed_password"),
+    salt (go.label = ""),
+    sugar (go.label)
```

Comment:
Can the items be upper-case (the labels can be lower-case)? That matches our
Thrift Style Guide.

    USERNAME (go.label = "surname")

And we should have one case where the label is a Thrift keyword since that's
the motivation behind this change.

Reply:
good advice, enum name and its label is actually case sensitive, I added upper/lower cases for item name and label, also added one reserved keyword label